### PR TITLE
Update test suite to pass on latest nightly

### PR DESCRIPTION
**Pull request by jkrems**
The port export [no longer exists](https://github.com/nodejs/node-inspect/pull/26) and isn't in use. It was a hold-over from the old built-in debugger.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "pretest": "eslint --rulesdir=tools/eslint-rules lib test",
-    "test": "tap \"test/**/*.test.js\"",
+    "test": "tap \"test/cli/*.test.js\"",
```
Comment (jkrems) [inline package.json]: This glob is interpreted differently, depending on who is running our test suite. This means that if we ever add tests outside of `test/cli` we'll have to change this.

**Comment by jkrems** [comment]
CI: https://ci.nodejs.org/job/node-inspect-continuous-integration/26/

**Comment by jkrems** [comment]
Woah, guess a lot changed in master recently.

**Comment by jkrems** [comment]
The new `Break on start` feature works great - but it stops at the function wrapper, not on the first line of the function wrapper. So everywhere we start or restart, we need to skip it.

**Comment by jkrems** [comment]
Passing CI against latest nightly: https://ci.nodejs.org/job/node-inspect-continuous-integration/28/

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "pretest": "eslint --rulesdir=tools/eslint-rules lib test",
-    "test": "tap \"test/**/*.test.js\"",
+    "test": "tap test",
```
Comment (dbushong) [inline package.json]: Is it ok that this might load non-test js files?  (ones intended to be require()ed from elsewhere?)

Comment (jkrems) [inline package.json]: Right now there's only one of those (`test/cli/start-cli.js`) which is a bit annoying. In future, I'd rather move those helpers into a sibling directory (`test-helpers/`) than risk the inconsistent glob behavior.

**Pull request merged**